### ISSUE TITLE
Snapshots of old vocabulary hierarchies (HTML format)

### ISSUE THREAD

**Issue by keighrim** [lapps/vocabulary-pages#63]
Maybe a good idea to keep old version(s) of vocab hierarchies for future reference. 
Something like https://vocab.lappsgrid.org/old/x.y.z .
These URI can then be used in LIF to indicate the version of vocabulary a tool used to generate the file. (See https://github.com/lapps/org.lappsgrid.serialization/issues/43#issuecomment-367820827) .

**Comment by marcverhagen** [comment]
Yes! Which means at least updating `templates/vocab-index.template`. And when creating a new vocabulary we should either retire the previous one under a version number or create the repositories always with version numbers and have vocab.lappsgrid.org point at the latest:

```
http://vocab.lappsgrid.org/
http://vocab.lappsgrid.org/1.1.0
http://vocab.lappsgrid.org/1.2.0
```
